download_images_this_person_does_not_exist.py

```
@@ -0,0 +1,27 @@
+            print(f"Image saved to: {file_path}")
+        else:
+            print(f"Error downloading image: {response.status_code}")
+    except Exception as e:
+        print(f"Error downloading image: {e}")
+
+if __name__ == "__main__":
+    # Create the folder if it doesn't exist
+    folder_path = os.path.join(os.path.dirname(__file__), "photos")
+    os.makedirs(folder_path, exist_ok=True)
+    # Download images in multi threading
+    for i in range(10):
```

Ruff (0.8.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)